packages/block-library/src/post-template/index.php

```diff
@@ -62,7 +62,7 @@ function render_block_core_post_template( $attributes, $content, $block ) {
 			)
 		)->render( array( 'dynamic' => false ) );
 		$post_classes  = esc_attr( implode( ' ', get_post_class( 'wp-block-post' ) ) );
```

Comment:
Since we're moving escaping, it should be removed from this line 
```suggestion
		$post_classes  = implode( ' ', get_post_class( 'wp-block-post' ) );
```

Reply:
Yeh let's avoid double escaping! 👍